cockatrice/src/carddatabasemodel.h

```diff
@@ -12,7 +12,7 @@ class FilterTree;
 class CardDatabaseModel : public QAbstractListModel {
     Q_OBJECT
 public:
-    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
+    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
```

Comment:
Annoyance > Use

(Plus, we have very nice filters for more specific use cases, which extend the basics that we offer in a very clean and user-friendly way to the everyday/new user...)


Imagine somebody is playing a format like Tiny Leaders... Nobody would think about ordering by cmc and scrolling to find suitable cards in a list of which 70% of all cards are not needed. Now think about a format where you are just allowed to use cmc 5,6 and 9 for example... everybody would just use the filters because it's way more convenient!
Or if you want to seriously brew and deeply fine tune your decks curve for example... you're going to use filters anyway. ;)

Another one. Let's say you want to simply build a new deck. You start your normal deck building process to build a RB Aggro for example. Now you order by the new color column to find yourself with colors ordered for BGRUW or WURGB. But no way black and red cards are all in one single list next to each other. *(this is an assumption)*
You don't think about filters here? ;)

So there is very few this would actually help users with I guess.

---

We can have all columns in the world, but then it needs to be customizable imo.
Scrolling something away is a very bad workaround for a new feature of questionable use for the majority in the first place.

Example of a right click on a column in the mail list of Thunderbird:
![columns](https://cloud.githubusercontent.com/assets/9874850/16313691/ac923356-397a-11e6-860d-5bd557b19db7.png)